examples/e-commerce/src/widgets/Configuration.js

```diff
@@ -1,7 +1,7 @@
 import { configure } from 'instantsearch.js/es/widgets';
 
 const configuration = configure({
-  attributesToSnippet: ['description'],
+  attributesToSnippet: ['name:10', 'description:10'],
```

Comment:
We want smaller titles and descriptions that take about two lines.

Reply:
With the snippet on the `name` attribute we might have the `...` at the beginning. It feels a bit weird for a product name.

![Screenshot 2019-06-05 at 14 47 49](https://user-images.githubusercontent.com/6513513/58957442-e64f8900-87a0-11e9-9ed2-cd71dff6dfd8.png)

Reply:
That's true, we might want to only use a snippet for the description.